File: 1544-make-the-string-great/1544-make-the-string-great.cpp
class Solution {
public:
    bool isbad(char x, char y) {
        if ((x - y == 32 ) || (x - y == -32))
            return true;
        return false;
    }
    string makeGood(string s) {
        int i = 0;
        int t = s.length() - 1;
        
        while (i < t)
        {
            if (isbad(s[i],s[i+1]))
            {
                s.erase(i,2);
                i = -1;
                t -= 2;
            }
            i++;
        }
                 
        return s;
        
    }
};

File: 1544-make-the-string-great/1544-make-the-string-great.py
class Solution:
    def isbad(self, x:str, y:str) -> bool:
        if ((ord(x) - ord(y)) in [-32,32]):
            return True
        return False
    
    def makeGood(self, s: str) -> str:
        t = len(s) - 1
        i = 0
        count = 0
        
        while(i < t):
            if (self.isbad(s[i],s[i+1])):
                s = s[:i] + s[i+2:]
                t -= 2
                i = -1
            i += 1
            
        return s